lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1928,7 +1928,13 @@ class SemanticsObject {
   void dispose() {
     assert(!_isDisposed);
     _isDisposed = true;
-    element.remove();
+
+    EnginePlatformDispatcher.instance.viewManager.safelyBlurElement(
+      element,
```

Comment:
`element` is not always the target of the focus. See, for example, https://github.com/flutter/engine/blob/656117fc8ba96b3a2d0a46711dbddec6bf44fcaf/lib/web_ui/lib/src/engine/semantics/label_and_value.dart#L81.

Maybe what we need is to have an abstract getter `DomElement get focusTarget` on the `SemanticRole` class, and then pass `role.focusTarget` to `safelyBlurElement`.